go.mod

```
@@ -2,4 +2,4 @@ module github.com/sosukesuzuki/node-actionlint
 
 go 1.16
 
-require github.com/rhysd/actionlint v1.6.8
+require github.com/rhysd/actionlint v1.6.13
```

```suggestion
require github.com/rhysd/actionlint v1.6.25
```
Much has changed since then, patch 13 will not properly validate the current schema

---

Thanks for your suggestion. Accept it.